cfgen/cfgen

```diff
@@ -831,22 +844,24 @@ def oids_to_dhall(oids: List[Oid]) -> str:
 NetProtocol = Enum('NetProtocol', 'tcp o2ib')
 
 
-ProcT = Enum('ProcT', 'hax m0_server m0_client_s3 m0_client_other')
+ProcT = Enum('ProcT', 'hax m0_ios m0_client_s3 m0_client_other')
 ProcT.__doc__ = 'Type of process'
 
 
 class LnetPortalGroup(IntEnum):
     hax = 1,
-    m0_server = 2,
+    m0_ios = 2,
     m0_client_s3 = 3,
     m0_client_other = 4
 
 
 class LibfabPortalGroup(IntEnum):
-    hax = 2000,
-    m0_server = 3000,
-    m0_client_s3 = 4000,
-    m0_client_other = 5000
+    hax = 22000,
+    m0_ios = 21000,
+    m0_client_s3 = 22500,
+    # Below one parameter consider for future refrences
```

Comment:
I think we need not have a code required for future reference and commented. Let's uncomment it, when we have the relevant service type it will be used automatically, or let's delete it for now.